network/glewlwyd/glewlwyd.SlackBuild

```diff
@@ -23,7 +23,7 @@
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=glewlwyd
-VERSION=${VERSION:-2.6.2}
+VERSION=${VERSION:-2.7.0}
```

Comment:
man pages are being installed in the wrong directory:

```sh
# tree --charset ascii usr/share/man/
usr/share/man/
`-- man8
    `-- glewlwyd.8

1 directory, 1 file
```

Reply:
Fixed